tests/test_core.py

```diff
@@ -81,7 +90,10 @@ def test_category_pass():
 
 
 def test_load_metadata(test_ia):
-    test_ia.load_metadata(os.path.join(here, 'testing_metadata.xlsx'))
+    with pytest.warns(Warning) as record:
```

Comment:
might be worth testing what warning we actually get. Nice pattern, haven't used it before =)